### ISSUE TITLE
Use confy to load `EdTheme`

### ISSUE THREAD

**Issue by Anton-4** [roc-lang/roc#2531]
We have some confy stuff commented out in some places, this should be re-enabled and themes should be able to be specified in a config file.

**Comment by skyqrose** [comment]
Editor is no longer being worked on, and has been removed from the repo. Closing.
